modules/PROGENY.py

```diff
@@ -0,0 +1,288 @@
+    'KIRP': {'uri': 'http://www.ebi.ac.uk/efo/EFO_0000000', 'label': 'kidney renal papillary cell carcinoma'},
+    'COREAD': {'uri': 'http://www.ebi.ac.uk/efo/EFO_0000000', 'label': 'colorectoral adenoma'}
+}
+
+''' TCGA acronyms '''
+TUMOR_TYPE_MAP = {
+    'BLCA': 'bladder carcinoma',
+    'BRCA': 'breast carcinoma',
+    'HNSC': 'head and neck squamous cell carcinoma',
+    'KIRC': 'clear cell renal carcinoma',
+    'LIHC': 'hepatocellular carcinoma',
+    'LUAD': 'lung adenocarcinoma',
+    'LUSC': 'squamous cell lung carcinoma',
+    'PRAD': 'prostate adenocarcinoma',
+    'STAD': 'stomach adenocarcinoma',
+    'THCA': 'thyroid carcinoma',
+    'UCEC': 'endometrial endometrioid adenocarcinoma',
+    'KICH': 'kidney chromophobe',
+    'KIRP': 'kidney renal papillary cell carcinoma',
+    #TODO tumor description need update
+    'COREAD': ''
+}
+
+''' Pathway -> Perturbed Targets
+    https://drive.google.com/drive/folders/1L5Y_umEZiccWJnXiiaYMNKUKYTjnp3ZU
+'''
+PATHWAY_TARGET_MAP = {
+    'Androgen' : ['AR'],
+    'EGFR'     : ['EGFR'],
+    'Estrogen' : ['ESR1'],
+    'Hypoxia'  : ['HIF1A'],
+    'JAK.STAT' : ['JAK1', 'JAK2', 'STAT1', 'STAT2'],
+    'MAPK'     : ['MAPK2K1', 'MAP2K2', 'RAF1'],
+    'NFkB'     : ['TLR4', 'NKFB1', 'RELA'],
+    'PI3K'     : ['PIK3CA', 'PI3K(Class1)'],
+    'TGFb'     : ['TGFBR1', 'TGFBR2'],
+    'TNFa'     : ['TNFRSF1A'],
+    'Trail'    : ['TNFSF10', 'BCL2', 'BCL-XL', 'BCL-W', 'MCL1'],
+    'VEGF'     : ['VEGFR'],
+    'WNT'      : ['WNT3A', 'GSK3A', 'GSK3B'],
+    'p53'      : ['TP53']
+}
+
+''' Pathway -> Reactome Pathway ID '''
+#TODO Hypoxia, JAK.STAT, Trail to be updated
+PATHWAY_REACTOME_MAP = {
+    'Androgen' : 'R-HSA-8940973:RUNX2 regulates osteoblast differentiation',
+    'EGFR'     : 'R-HSA-8856828:Clathrin-mediated endocytosis',
+    'Estrogen' : 'R-HSA-8939902:Regulation of RUNX2 expression and activity',
+    'Hypoxia'  : 'R-HSA-123456: XXX Pathway desc to be updated',
+    'JAK.STAT' : 'R-HSA-6785807:Interleukin-4 and 13 signaling',
+    'MAPK'     : 'R-HSA-2559580:Oxidative Stress Induced Senescence',
+    'NFkB'     : 'R-HSA-9020702:Interleukin-1 signaling',
+    'PI3K'     : 'R-HSA-8853659:RET signaling',
+    'TGFb'     : 'R-HSA-2173788:Downregulation of TGF-beta receptor signaling',
+    'TNFa'     : 'R-HSA-5357956:TNFR1-induced NFkappaB signaling pathway',
+    'Trail'    : 'R-HSA-123456:XXX Pathway desc to be updated',
+    'VEGF'     : 'R-HSA-1234158:Regulation of gene expression by Hypoxia-inducible Factor',
+    'WNT'      : 'R-HSA-381340:Transcriptional regulation of white adipocyte differentiation',
+    'p53'      : 'R-HSA-2559580:Oxidative Stress Induced Senescence'
+}
+
+''' These symbols are secondary/generic/typo that needs update '''
+PROGENY_SYMBOL_MAPPING = {
+    'NKFB1'   : 'NFKB1',
+    'MAPK2K1' : 'PRKMK1',
+    'PI3K(Class1)' : 'PIK3CA',
+    'VEGFR'   : 'KDR',
+    'BCL-W'   : 'BCL2L2',
+    'BCL-XL'  : 'BCL2L1'
+}
+
+class PROGENY():
```

Comment:
as above, I am not sure if you need to have a class that runs the code.
all of the below could be contained in a `main()` method.
You need classes only when you want to make multiple independent instances containing different states. For example `evidenceStrings`, as you do below.